cmake/developer_package/frontends/frontends.cmake

```diff
@@ -141,8 +141,13 @@ macro(ov_add_frontend)
     # Disable all warnings for generated code
     set_source_files_properties(${PROTO_SRCS} ${PROTO_HDRS} PROPERTIES COMPILE_OPTIONS -w GENERATED TRUE)
 
+    # Shutdown protobuf when unloading the front dynamic library
+    if(OV_FRONTEND_SHUTDOWN_PROTOBUF AND BUILD_SHARED_LIBS)
+        set(SHUTDOWN_PROTOBUF ${PROJECT_SOURCE_DIR}/src/frontends/common/shutdown_protobuf/shutdown_protobuf.cpp)
```

Comment:
can we create `INTERFACE` library in cmake and add these sources as `PUBLIC` ones (via https://cmake.org/cmake/help/latest/command/target_sources.html)? 

In this case we can use:
```cmake
target_link_libraries(${TARGET_NAME} PRIVATE ov_protobuf_shutdown)
```
`CMakeLists.txt` with such code can be in `https://github.com/meiyang-intel/openvino/tree/meiyang/fix_protobuf_memory_leak/src/frontends/common/shutdown_protobuf/CMakeLists.txt`

Reply:
have updated